src/Explorer/Tabs/TabsBase.ts

```diff
@@ -143,6 +142,26 @@ export default class TabsBase extends WaitsForTemplateViewModel {
     return (this.collection && this.collection.container) || (this.database && this.database.container);
   }
 
+  public getTitle(options: ViewModels.TabOptions): string {
+    const coll = this.collection?.id();
+    const db = this.database?.id();
+    if (coll) {
+      if (coll.length > 8) {
+        return coll.slice(0, 5) + "...." + options.title;
```

Comment:
An ellipsis has 3 dots and I'd recommend using the ellipsis character instead of actual 3 dots: "…" (Alt+0133)

now:
![image](https://github.com/Azure/cosmos-explorer/assets/951587/d659fdf0-74f2-4d17-9d1a-12e64ac27253)

with an ellipsis (character):
![image](https://github.com/Azure/cosmos-explorer/assets/951587/f2c0e273-48dc-4d57-b093-14ae8012a47c)

But overall looks much nicer now.

Reply:
Thanks, I just noticed the 4 vs 3 dots myself.